panther_driver/src/driver_node.py

```diff
@@ -1,24 +1,29 @@
 #!/usr/bin/python3
 
+import canopen
 import math
-
 from numpy import NaN
+import yaml
+
 import rospy
-import canopen
-from sensor_msgs.msg import BatteryState
-from sensor_msgs.msg import JointState
-from geometry_msgs.msg import Twist
+import tf2_ros
+
 from geometry_msgs.msg import Pose
 from geometry_msgs.msg import TransformStamped
+from geometry_msgs.msg import Twist
 from nav_msgs.msg import Odometry
-import tf2_ros
-from PantherKinematics import PantherKinematics
+from sensor_msgs.msg import BatteryState
+from sensor_msgs.msg import JointState
+
 from ClassicKinematics import PantherClassic
 from MecanumKinematics import PantherMecanum
 from MixedKinematics import PantherMix
-import yaml
 
 
+def sign(x):
+    x_sign = bool(x > 0) - bool(x < 0)
+    return x_sign if x_sign != 0 else 1
+
```

Comment:
```suggestion
def sign(x):
    return bool(x >= 0) - bool(x < 0)
```
This way should be simpler